tracer/build/_build/CodeGenerators/TargetFrameworks.cs

```diff
@@ -27,3 +27,12 @@ internal enum TargetFrameworks : uint
     NET8_0 = 512,
     NET0_0 = 1024,
```

Comment:
Shouldn't this be `NET9_0`? 😖

Reply:
Yep, removed in the simplification process